src/runtime/meter-toast-api.js

```diff
@@ -160,6 +172,26 @@ export class MeterToastApi {
     });
   }
 
+  /**
+   * Changes the size of the loading iframe on desktop to match the size of
+   * the meter toast iframe.
+   */
+  setLoadingViewWidth_() {
+    const mq = this.win_.matchMedia(
+      '(min-width: 640px) and (min-height: 640px)'
+    );
+    const element = this.dialogManager_
```

Comment:
This var could move inside the if statement

Reply:
Done